src/interface/src/app/treatments/treatments-routing-data.ts

```diff
@@ -5,6 +5,7 @@ export interface TreatmentRoutingData extends Data {
   scenarioId: number;
   treatmentId: number;
   projectAreaId?: number;
+  planId?: number;
```

Comment:
lets make this required- we should always have planId given our routing structure

Reply:
Updated, you will see it in the next commit